src/colors/themeColors.ts

```diff
@@ -0,0 +1,43 @@
+type ColorShade = {
+  main: string;
+  dark: string;
+  light: string;
+};
+
+type ThemeColors = {
+  primary: ColorShade;
+  secondary: ColorShade;
+  error: ColorShade;
+  warning: ColorShade;
+  success: ColorShade;
+};
+
+const themeColors: ThemeColors = {
```

Comment:
Add the missing properties that you didn't pass:
``` ts
mode: 'light',
text: {
    primary: '#333333',
    secondary: '#5c5c5c',
    disabled: '#b4b4b4',
  },
```
PLEASE CHECK IF ALL OF THESE IS RIGHT AND I DIDN'T LEFT ANYTHING BEHIND